src/features/events/tests/groupedEventsCard.test.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import { GroupedEventsCards } from '../index'
+import { mockNodes, testEvent } from './__mocks__'
+import 'jest-styled-components'
+
+describe('The GroupedEventsCards component', () => {
+  describe('Render', () => {
+    const wrapper = shallow(
+      <GroupedEventsCards
+        event={testEvent}
+        index={1}
+        events={mockNodes}
+        toLoad={24}
+      />
+    )
+
+    it('Snapshot', () => {
+      expect(wrapper).toMatchSnapshot()
+    })
+
+    describe('Props', () => {
+      it('passing event.node correctly', () => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not pass arrow functions to it()](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4658697)